v3/README.md

```diff
@@ -0,0 +1,40 @@
+# Buildpack v3 reference implementation
+
+Execution of these images differs from the standard packs API.
+
+Optionally, you may want to create a Docker volume to share files across runs:
```

Comment:
Probably want to add instructions for the export/analyze images as well. The `pack` CLI will be able to coordinate all of this very soon though.

Reply:
will do. how far away is "very soon"? these commands are complicated enough that i will forget them :)